projects/components/select/src/select.component.ts

```diff
@@ -343,7 +350,7 @@ export class PsSelectComponent<T = unknown> extends _PsSelectMixinBase
 
   private _propagateValueChange(value: any, source: ValueChangeSource) {
     this._value = value;
-    this.empty = this.multiple ? !value?.length : value == null;
+    this.empty = this.multiple ? !value?.length : !value;
```

Comment:
please add a test for the bug this fixes

Reply:
done